.renovaterc.json

```diff
@@ -50,6 +50,17 @@
         "golang/go"
       ],
       "groupName": "golang"
+    },
+    {
+      "description": "Group golangci-lint related bumps",
+      "matchPackageNames": [
+        "catalog/golang/golangci-lint",
+        "radiofrance/lint-config"
+      ],
```

Comment:
```suggestion
      "matchPackageNames": [
        "golangci/golangci-lint",
        "golangci/golangci-lint-action",
        "radiofrance/lint-config"
      ],
```

Reply:
@Thibaut-gauvin I don't think adding `golangci/golangci-lint-action` is necessary, as the golangci-lint version is already defined on this line:
https://github.com/radiofrance/dib/blob/6e823891cd78c3bf6a3f141acc8c1c7659c118d8/.github/workflows/qa.yml#L35

Reply:
I added it anyway!